pkg/cmd/shared/flags_handler.go

```diff
@@ -0,0 +1,40 @@
+package handler
+
+import (
+	"github.com/algolia/cli/pkg/cmd/synonyms/shared"
+	"github.com/spf13/cobra"
+)
+
+type FlagsHandler interface {
+	Validate() error
+	AskAndFill() error
+}
+
+// Synonyms
```

Comment:
I would rather have all the specific logic and methods related to the synonyms inside the `synonyms` folder, WDYT?

Reply:
I saw that file to contains logic and methods for all commands that'll follow this pattern (rules, objects...) in the future
I wanted to follow that example: https://gobyexample.com/interfaces because they'll have the same methods to implement (`Validate` and `AskAndFill`, triggered by `HandleFlags`)

I may be wrong or unclear, I can change that if it's the case 🙇